src/app/starredentries/starredentries.component.html

```diff
@@ -81,12 +82,42 @@
           </div>
         </mat-tab>
 
-        <ng-template #workflowSummary let-workflow="workflow" let-isAppTool="isAppTool">
+        <mat-tab>
+          <ng-template mat-tab-label>
+            <div fxLayoutGap="0.5rem">
+              <img class="notebook-icon" src="../../../assets/svg/sub-nav/notebooks.svg" alt="notebook logo" />
+              <span>Notebooks</span>
+              <span class="tab-display" data-cy="starred-notebooks-count">{{ starredNotebooks?.length }}</span>
+            </div>
+          </ng-template>
+          <div>
+            <mat-card class="alert alert-info starred-header" role="alert" *ngIf="starredNotebooks?.length === 0">
```

Comment:
This banner needs to also have `mat-elevation-z` added to the class so that it matches the banner style of the redesign. (Last point of the UI design section https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#ui-design)